docs/howtos/istio.md

```diff
@@ -1,330 +1,402 @@
-
-You can see if the two Ambassador Edge Stack services are running correctly (and also obtain the LoadBalancer IP address when this is assigned after a few minutes) by executing the following commands:
-
-```shell
-$ kubectl get services
-NAME               TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)          AGE
-ambassador         LoadBalancer   10.63.247.1     35.224.41.XX     8080:32171/TCP     11m
-ambassador-admin   NodePort       10.63.250.17    <none>           8877:32107/TCP   12m
-details            ClusterIP      10.63.241.224   <none>           9080/TCP         16m
-kubernetes         ClusterIP      10.63.240.1     <none>           443/TCP          24m
-productpage        ClusterIP      10.63.248.184   <none>           9080/TCP         16m
-ratings            ClusterIP      10.63.255.72    <none>           9080/TCP         16m
-reviews            ClusterIP      10.63.252.192   <none>           9080/TCP         16m
-
-$ kubectl get pods
-NAME                             READY     STATUS    RESTARTS   AGE
-ambassador-2680035017-092rk      2/2       Running   0          13m
-ambassador-2680035017-9mr97      2/2       Running   0          13m
-ambassador-2680035017-thcpr      2/2       Running   0          13m
-details-v1-3842766915-3bjwx      2/2       Running   0          17m
-productpage-v1-449428215-dwf44   2/2       Running   0          16m
-ratings-v1-555398331-80zts       2/2       Running   0          17m
-reviews-v1-217127373-s3d91       2/2       Running   0          17m
-reviews-v2-2104781143-2nxqf      2/2       Running   0          16m
-reviews-v3-3240307257-xl1l6      2/2       Running   0          16m
-```
-
-Above we see that external IP assigned to our LoadBalancer is 35.224.41.XX (XX is used to mask the actual value), and that all ambassador pods are running (Ambassador Edge Stack relies on Kubernetes to provide high availability, and so there should be two small pods running on each node within the cluster).
-
-You can test if the Ambassador Edge Stack has been installed correctly by using the test route to `httpbin.org` to get the external cluster [Origin IP](https://httpbin.org/ip) from which the request was made:
+- [Istio 1.5 and above](#istio-1.5-and-above)
+- [Istio 1.4 and below](#istio-1.4-and-below)
 
-```shell
-$ curl -L 35.224.41.XX/httpbin/ip
-{
-  "origin": "35.192.109.XX"
-}
-```
+### Istio 1.5 and Above
 
-If you're seeing a similar response, then everything is working great!
+Istio 1.5 introduced [istiod](https://istio.io/docs/ops/deployment/architecture/#istiod) which moved Istio away from a microservice architecture and towards a single control plane process. 
```

Comment:
I wouldn’t say it moved away from a micro service architecture.  I’d just say single control plane process.